### PR TITLE
Add Mayavi and FSLeyes migration to osx-arm64

### PR DESCRIPTION
**Pull request by hoechenberger**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/migrations/osx_arm64.txt [hunk 1]
```diff
@@ -73,9 +73,12 @@ wrapt
 mne-base
 mne
 mne-qt-browser
+mne-installer-menus
```
Comment (dopplershift) [inline recipe/migrations/osx_arm64.txt:76]: I don't understand...this one is noarch?

Comment (hoechenberger) [reply recipe/migrations/osx_arm64.txt:76]: I'm sorry, you are right – I will remove this one from the PR!

Comment (dopplershift) [reply recipe/migrations/osx_arm64.txt:76]: To be clear, based on that and its dependencies, I'd be shocked to find out this is not installable on osx arm64 right now.

Comment (hoechenberger) [reply recipe/migrations/osx_arm64.txt]: I just checked, and indeed it can be installed on `osx-arm64` :)

**Comment by hoechenberger** [comment]
Thanks, @dopplershift!

**Pull request merged**